### PR TITLE
Initial package.json, CircleCI

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

**Comment by akoushke** [comment]
testing

**Comment by akoushke** [comment]
testing

**Comment by akoushke** [comment]
test

**Comment by akoushke** [comment]
testing

**Comment by akoushke** [comment]
test

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  install:
+    executor: main-executor
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - dependency-cache-{{ checksum "package-lock.json" }}
+      - run:
+          name: Install Dependencies
+          command: npm i
```
Comment (adamweeks) [inline .circleci/config.yml]: Didn't the team discover some issues with this type of setup and moved over to `npm ci` last week?

**Comment by adamweeks** [comment]
Would like to see a few changes to the commit messages to follow the contributing style from the other repos:

`feat(package): init` to `feat(package): add initial package.json`
`feat(git): add .npmignore` to `build: add .npmignore`
`build(CirclCI): init` to `ci: add initial circleci config`

File: .circleci/config.yml (hunk 1 above)
Comment (akoushke) [reply .circleci/config.yml]: not sure if `npm ci` works with our logic here.  we don't want to remove the modules if the `package-lock.json` had not been updated. therefore `npm i` is what we need.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "publishConfig": {
+    "tag": "latest",
+    "access": "public"
+  },
+  "author": "devsupport@webex.com",
+  "license": "MIT",
+  "dependencies": {
+    "@webex/components": "^1.35.0",
+    "@webex/sdk-component-adapter": "^1.19.3",
+    "webex": "^1.80.103"
```
[the remark this answers is not in the document]
Comment (akoushke) [reply package.json]: why?

[the remark this answers is not in the document]
Comment (lalli-flores) [reply package.json]: Because we have a specific version of the SDK we wanted to use with the SDK adapter

[the remark this answers is not in the document]
Comment (akoushke) [reply package.json]: I'm not following. what is the problem of using the latest SDK?

[the remark this answers is not in the document]
Comment (lalli-flores) [reply package.json]: Why did we hard code the SDK version in the adapter if there is no issues with new versions then?

[the remark this answers is not in the document]
Comment (akoushke) [reply package.json]: I have no idea. I have not ran into any problem using the latest SDK version with our adapter.

File: .npmignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+*
```
[the remark this answers is not in the document]
Comment (akoushke) [reply .npmignore]: will be addressed with `semantic-release` PR

**Comment by lalli-flores** [comment]
> Would like to see a few changes to the commit messages to follow the contributing style from the other repos:
> 
> `feat(package): init` to `feat(package): add initial package.json`
> `feat(git): add .npmignore` to `build: add .npmignore`
> `build(CirclCI): init` to `ci: add initial circleci config`

I would suggest even renaming all `feat` scopes. All of these changes are `build` mostly

File: .npmignore (hunk 3 above)
[the remark this answers is not in the document]
Comment (lalli-flores) [reply .npmignore]: I'm ok with that, as long as we don't commit a `.npmignore` file 👍🏻

**Pull request merged**